app/helpers/draw_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+#
+# Helper module for Draws
+module DrawHelper
+  # Check to see if a draw is startable by the current user. Returns true if the
+  # draw is in the 'draft' status and the user satisfies the Pundit policy,
+  # false otherwise.
```

Comment:
Ha, thanks for catching that. I forgot we could do more than assert on `user` in policies 😛. I'll update.